src/Week04/HW04.hs

```diff
@@ -0,0 +1,98 @@
+{-# LANGUAGE InstanceSigs #-}
+{-# OPTIONS_GHC -Wall #-}
```

Comment:
This is redundant, as we have -Wall enabled for all files in the project (see -Wall occurrences in the cabal file).
```suggestion
```